Lib/test/libregrtest/main.py

```diff
@@ -458,6 +466,9 @@ def get_tests_result(self):
             result.append("FAILURE")
         elif self.ns.fail_env_changed and self.environment_changed:
             result.append("ENV CHANGED")
+        elif not any((self.good, self.bad, self.skipped, self.interrupted,
```

Comment:
What do you think of always appending "NO TEST RUN" if run_no_tests is non-empty?

"./python -m test -j0 test_os test_sys -m test_access" would say "Tests result: SUCCESS, NO TEST RUN".

Does it sound weird to you?

My expectation is that "SUCCESS, NO TEST RUN" would be rare. You would only get it if you pass --match. But if you get it, it means that *maybe* you did a mistake. That's why the exit code is 0 (success) and not 1 (error). It's just a hint, not an hard error.

Reply:
Hummmm..... I think is a good idea. What do you think about somethink on the lines of:

```python
❯ ./python.exe -m test -j0 test_os test_sys -m test_access
Run tests in parallel using 10 child processes
0:00:00 load avg: 1.72 [1/2] test_sys run no tests
0:00:00 load avg: 1.72 [2/2] test_os passed

== Tests result: SUCCESS, (SOME TESTS DID NOT EXECUTE ANY SUBTESTS) ==

1 test OK.

1 test run no tests:
    test_sys

Total duration: 473 ms
Tests result: SUCCESS, (SOME TESTS DID NOT EXECUTE ANY SUBTESTS)
````

only if the output is SUCCESS? So basically:

```python
if not result:
     result.append("SUCCESS")
     if self.run_no_tests:
         result.append("(SOME TESTS DID NOT EXECUTE ANY SUBTESTS)")
```

Reply:
> only if the output is SUCCESS

I don't think that it's useful. And I prefer "NO TEST RUN" because it's shorter :-)

Reply:
@pablogsal: what do you think? ping?

Reply:
Sorry for the late response.

I agree, I think printing "NO TEST RUN" when there is at least one item in `run_no_tests` is a good idea :)

I implemented it in 8a64c0b

Reply:
Wait, I am checking somethig that is not fully correct with that commit.

Reply:
As we are setting "SUCCESS" only when there are no other thing in `result` this means that running `./python -m test -j0 test_os -m test_no_existing` will also result in:

Tests result: SUCCESS, NO TEST RUN

I suposse that is not what we want...

Reply:
> As we are setting "SUCCESS" only when there are no other thing in `result` this means that running `./python -m test -j0 test_os -m test_no_existing` will also result in:
> 
> Tests result: SUCCESS, NO TEST RUN
> 
> as to achieve the result for `"./python -m test -j0 test_os test_sys -m test_access"` we would need to append after the "SUCCESS" is added:
> 
> ```
> if not result:
>     result.append("SUCCESS")
> if self.run_no_tests:
>     result.append("NO TEST RUN")
> ```
> otherwise, you never will get "SUCCESS" when "self.run_no_test" is populated.
> 
> I suposse that is not what we want. What do you think?

Ok, nervermind. I implemented it in 2272ba8 so only `Tests result: SUCCESS, NO TEST RUN` will happen if there is at least one test that succeeds and some did not run but only `NO TEST RUN` appears if no test at all have ran. Some examples:

```
./python -m test -j0 test_os test_sys -m test_access
Tests result: SUCCESS, NO TEST RUN

./python -m test -j0 test_os test_sys -m test_blech
Tests result: NO TEST RUN

./python -m test -j0 test_os -m test_blech
Tests result: NO TEST RUN

./python -m test -j0 test_os -m test_access
Tests result: SUCCESS
```